### ISSUE TITLE
test1.hpp fails to build

### ISSUE THREAD

**Issue by ajknapp** [odinthenerd/abcd#6]
The file test1.hpp fails to build with the following error.

```
In file included from /home/andy/tmp/abcd/test/test.cpp:1:0:
/home/andy/tmp/abcd/test/test1.hpp: In function ‘int test1::run()’:
/home/andy/tmp/abcd/test/test1.hpp:105:11: error: ‘class kvasir::abcd::combiner<test1::my_policy, test1::other_policy, test1::my_allocator>’ has no member named ‘foo’
         m.foo();
           ^~~
/home/andy/tmp/abcd/test/test1.hpp:106:11: error: ‘class kvasir::abcd::combiner<test1::my_policy, test1::other_policy, test1::my_allocator>’ has no member named ‘bar’
         m.bar();
           ^~~
```

The example in README.md is also out of date. I've updated it to the extent possible, but it fails with an analogous error. I'll send a pull request once the missing member issue is resolved.

**Comment by odinthenerd** [comment]
thanks a lot! I have been neglecting this a bit due to time constraints

**Comment by ghost** [comment]
Fixed in #8 and thank you.
